corehq/apps/reports/tests/readable_forms/top_level_refless_group.result.yaml

```diff
@@ -764,14 +765,12 @@
   relevant: null, repeat: null, required: null, response: all, tag: null, type: null,
   value: /progress_note/note/adherence/self_report}
 - {calculate: null, comment: null, group: null, label: note/conversations/which, relevant: null,
-  repeat: null, required: null, response: '', tag: null, type: null, value:
-  /progress_note/note/conversations/which}
+  repeat: null, required: null, response: '', tag: null, type: null, value: /progress_note/note/conversations/which}
 - {calculate: null, comment: null, group: null, label: note/discussed_appointments,
   relevant: null, repeat: null, required: null, response: 'no', tag: null, type: null,
   value: /progress_note/note/discussed_appointments}
 - {calculate: null, comment: null, group: null, label: note/discussed_followup, relevant: null,
-  repeat: null, required: null, response: 'no', tag: null, type: null, value:
-  /progress_note/note/discussed_followup}
+  repeat: null, required: null, response: 'no', tag: null, type: null, value: /progress_note/note/discussed_followup}
 - {calculate: null, comment: null, group: null, label: note/discussed_goals, relevant: null,
   repeat: null, required: null, response: 'no', tag: null, type: null, value: /progress_note/note/discussed_goals}
 - {calculate: null, comment: null, group: null, label: note/discussed_qsp, relevant: null,
```

Comment:
@dannyroberts all these changes are noop changes due to ordering of dict keys. I didn't have time to dig into how to force the ordering. Do you happen to know?

This is generated here: https://github.com/dimagi/commcare-hq/blob/3e88dbe4554316981e7f9a43fde34aa8870702c9/corehq/apps/reports/tests/test_readable_formdata.py#L268

Reply:
I think that the line you referenced _does_ force ordering.

```
>>> print yaml.safe_dump([{'e': 5, 'f': 6, 'g': 7, 'a': 1, 'b': 2, 'c': 3, 'd': 4, 'A': 123, 'B': 123}])
- {A: 123, B: 123, a: 1, b: 2, c: 3, d: 4, e: 5, f: 6, g: 7}
```
Could it be that the last person to edit these files generated them in a different way?

Reply:
Thanks, sorry I clearly didn't look closely enough.

Reply:
👍 no worries